woocommerce-gateway-stripe.php

```diff
@@ -205,6 +205,7 @@ public function init() {
 				require_once dirname( __FILE__ ) . '/includes/admin/class-wc-stripe-upe-compatibility-controller.php';
 				require_once dirname( __FILE__ ) . '/includes/migrations/class-allowed-payment-request-button-types-update.php';
 				require_once dirname( __FILE__ ) . '/includes/class-wc-stripe-account.php';
+				require_once dirname( __FILE__ ) . '/includes/class-wc-stripe-link-payment-token.php';
```

Comment:
Can we relocate this include statement next to the SEPA token require statement?

Reply:
changed in [e010c7a](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2455/commits/e010c7af3141127f1cd9a7146ac986611708a664). thank you!